catroid/src/main/res/layout/landing_page.xml

```diff
@@ -46,7 +47,7 @@
 
 
                 <RelativeLayout
-                    android:id="@+id/current_project"
+                    android:id="@+id/currentProjectLayout"
```

Comment:
snakecase please for all variables in the xml

Reply:
pls see my comment below